plugins/poll/assets/stylesheets/common/poll.scss

```diff
@@ -28,6 +28,57 @@ div.poll-outer {
       }
     }
 
+    .irv-poll-options {
+      display: flex;
+      flex-direction: column;
+      gap: 0.5em;
+      padding: 0.5em;
+
+      .irv-poll-option {
+        display: inline-flex;
+        details {
+          padding: 0;
+        }
+        summary {
```

Comment:
can we style a class instead?

Reply:
turns out it's redundant, removed: https://github.com/discourse/discourse/pull/27155/commits/b67348cbf4bc3a1ab29ae421bee7352f7c96a018